pyemma/msm/tests/test_amm.py

```diff
@@ -0,0 +1,766 @@
+# This file is part of PyEMMA.
```

Comment:
That could be possible now, but maybe it is good to keep them separate as a lot of the functionality does not translate directly between the models. Also to make room for some divergence in future developments. What do you think?